domains/transfer/Transfer.API/API/API.csproj

```diff
@@ -67,4 +67,8 @@
       <ProjectReference Include="..\..\Shared\ProjectOriginClients\ProjectOriginClients.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Folder Include="Shared\Extensions\" />
+    </ItemGroup>
```

Comment:
Why this include?

Reply:
No idea actually. Must be something automatically added. i can remove it

Reply:
Please do :)